lib/src/drives.js

```diff
@@ -21,38 +21,8 @@
  * THE SOFTWARE.
  */
 
-var drivelist = require('drivelist');
-
-/**
- * @summary List all available drives
- * @function
- * @public
- *
- * @description
- * See https://github.com/resin-io/drivelist
- *
- * @fulfil {Object[]} - available drives
- * @returns {Promise}
- *
- * @example
- * drives.list().then(function(drives) {
- *   drives.forEach(function(drive) {
- *     console.log(drive.device);
- *   });
- * });
- */
-exports.list = function() {
-  'use strict';
-
-  return new Promise(function(resolve, reject) {
-    drivelist.list(function(error, drives) {
-      if (error) {
-        return reject(error);
-      }
-      return resolve(drives);
-    });
-  });
-};
+var Promise = require('bluebird');
```

Comment:
'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.